idaes/models/properties/modular_properties/phase_equil/tests/test_smooth_VLE_2.py

```diff
@@ -339,3 +343,81 @@ def test_calculate_ceos_derivative_slacks(frame):
 
     assert value(gp["Vap"]) == pytest.approx(EPS_INIT, abs=1e-8)
     assert value(gn["Vap"]) == pytest.approx(EPS_INIT, abs=1e-8)
+
+
+@pytest.fixture()
+def H2O_H2_model():
+    m = ConcreteModel()
+
+    # Create a dummy parameter block
+    m.params = GenericParameterBlock(
+        components={
+            "H2O": {
+                "parameter_data": {
+                    "pressure_crit": (220.6e5, pyunits.Pa),
+                    "temperature_crit": (647, pyunits.K),
+                    "omega": 0.344,
+                    "pressure_sat_comp_coeff": {  # NIST <- Stull 1947
+                        "A": 4.6543,
+                        "B": 1435.264,
+                        "C": -64.848,
+                    },
+                },
+                "pressure_sat_comp": NIST,
+                "phase_equilibrium_form": {("Vap", "Liq"): fugacity},
+            },
+            "H2": {
+                "valid_phase_types": [PhaseType.vaporPhase],
+                "parameter_data": {
+                    "pressure_crit": (13e5, pyunits.Pa),
+                    "temperature_crit": (33.2, pyunits.K),
+                    "omega": -0.218,
+                },
+            },
+        },
+        phases={
+            "Liq": {
+                "equation_of_state": Cubic,
+                "equation_of_state_options": {"type": CubicType.PR},
+            },
+            "Vap": {
+                "equation_of_state": Cubic,
+                "equation_of_state_options": {"type": CubicType.PR},
+            },
+        },
+        state_definition=FTPx,
+        pressure_ref=100000.0,
+        temperature_ref=300,
+        base_units={
+            "time": pyunits.s,
+            "length": pyunits.m,
+            "mass": pyunits.kg,
+            "amount": pyunits.mol,
+            "temperature": pyunits.K,
+        },
+        phases_in_equilibrium=[("Vap", "Liq")],
+        phase_equilibrium_state={("Vap", "Liq"): CubicComplementarityVLE},
+        parameter_data={
+            "PR_kappa": {
+                ("H2O", "H2O"): 0.000,
+                ("H2", "H2O"): 0.000,
+                ("H2O", "H2"): 0.000,
+                ("H2", "H2"): 0.000,
+            }
+        },
+    )
+
+    # Create a dummy state block
+    m.props = m.params.state_block_class([1], parameters=m.params)
+
+    return m
+
+
+@pytest.mark.unit
+def test_calculate_teq_permanent_gas(H2O_H2_model):
+    m = H2O_H2_model
+    m.props[1].pressure.set_value(1e5)
+    m.props[1].temperature.set_value(300)
+    CubicComplementarityVLE.calculate_teq(m.props[1], ("Vap", "Liq"))
+    assert not m.props[1].is_property_constructed("temperature_dew")
+    assert not m.props[1].is_property_constructed("temperature_bubble")
```

Comment:
It appears that the framework does not create these equilibrium properties because there is only one VLE component in the system, which makes sense. Are there tests for other scenarios, for example one or more non-condensable or non-volatile components in a system with two or more VLE components? In those cases, these properties should still be created.

Reply:
The `SmoothVLE2` method doesn't need bubble and dew points to operate, unlike the original `SmoothVLE`. The test coverage here is pretty poor, but I wanted to fix the outstanding issue quickly before moving onto other things.